lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1262,12 +1267,10 @@ def _rgb(self, im):
         rgba = np.fromstring(s, np.uint8)
         rgba.shape = (h, w, 4)
         rgba = rgba[::-1]
-        rgb = rgba[:, :, :3].tostring()
-        a = rgba[:, :, 3]
-        if np.all(a == 255):
+        rgb = np.ascontiguousarray(rgba[:, :, :3])
```

Comment:
I'm not sure, but it just basically calls `array(..., order='C')` with some additional arguments that don't matter here, so I used that instead.